tidb-cloud/cppo-customer.md

```diff
@@ -0,0 +1,32 @@
+---
+title: AWS Channel Partner Private Offer (CPPO) Customer
+summary: Learn how to become an AWS Channel Partner Private Offer (CPPO) customer.
+---
+
+# AWS Channel Partner Private Offer (CPPO) Customer
+
+An AWS Channel Partner Private Offer (CPPO) customer subscribes to TiDB Cloud services through a Channel Partner in the AWS Marketplace.
+
+Unlike direct TiDB Cloud customers, AWS CPPO customers pay invoices via the AWS Marketplace rather than directly to PingCAP. However, all other daily operations in the TiDB Cloud console remain the same for both AWS CPPO and direct TiDB Cloud customers.
```

Comment:
```suggestion
Unlike direct TiDB Cloud customers, reseller's customers pay invoices via the AWS Marketplace rather than directly to PingCAP. However, all other daily operations in the TiDB Cloud console remain the same for both AWS CPPO and direct TiDB Cloud customers.
```